### PR TITLE
Add PR Template

### PR DESCRIPTION
**Pull request by relyt4me**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+### Description
+*
+
+### Files modified:
+- [ ] web/src/....
+- [ ] 
+- [ ] Other files:
```
Comment (jordy1611) [inline .github/pull_request_template.md]: There's a few dozen files. Maybe it's easier just to type out files modified rather than pre listing them

**Comment by Leasw144** [comment]
Looks good to me

**Review by janegreene — APPROVED** [review]
Looks great, thank you for handling!

**Review by jordy1611 — APPROVED** [review]
Looks great thanks for working on it

**Pull request merged**